os/android/gralloc1bufferhandler.cpp

```diff
@@ -127,6 +130,19 @@ bool Gralloc1BufferHandler::CreateBuffer(uint32_t w, uint32_t h, int format,
 
   set_format_(gralloc1_dvc, temp->gralloc1_buffer_descriptor_t_, pixel_format);
 
+#ifdef ENABLE_RBC
```

Comment:
Do we need to use RBC for all those supported formats?
How about if if RBC is not support on the plane for display? It may impact performance?
As I know only primary plane can directly scanout compressed buffer